integration/nwo/fabric/topology/core_template.go

```diff
@@ -282,6 +282,14 @@ fabric:
         files:
         - {{ .PeerLocalTLSDir Peer }}/ca.crt
       rootCertFile: {{ .CACertsBundlePath }}
+      keepalive:
+       client:
+         interval: 60s
+         timeout: 600s
+       server:
+         interval: 60s
+         timeout: 600s
+         minInterval: 60s 
```

Comment:
Just thinking ... how would a FSC user be able to change these settings without touch `core_template.go`?

Reply:
just changing the configuration file, no?

Reply:
or we need explicit support on the topology. Let's wait for a first request on this 😅